outreach_announcement.md

```diff
@@ -1,15 +1,29 @@
+## Email blurb
+
+**Caution**: modify items in **bold** as needed before sending!
+
 Hi all,
 
-I wanted to let you know about the coding and open research Study Group recently started at the University of Toronto. The Mozilla Science Lab has started a series of meetup groups for scientists and researchers to get together, share coding skills, and help each other out with the programming nearly all of us have to do as part of our research. Since June, the U of T Study Group has met weekly, and we've covered topics in Python, Git, and R and discussed many other topics related to life as a researcher. 
+Are you interested in learning more about programming and tools for research? UofT Coders is a place for researchers and learners to get together, ask questions, share ideas and skills, and build community with others at the University of Toronto. UofT Coders meets every week on Wednesdays at 6 pm, year round. Everyone is welcome, regardless of coding experience or background.
 
-Most Study Group sessions are hour-long tutorials where one person from our community leads a hands-on demonstration of a tool, package or piece of code, while everyone else follows along. Also, we get together once in a while for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set. 
+Most UofT Coders sessions are hour-long tutorials where one person from our community leads a hands-on demonstration of a tool, package or piece of code, while everyone else follows along. We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set. 
```

Comment:
"We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set."

->

"These tutorials are led by volunteers, and our members design the curriculum and schedule. We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise."